Sources/CodeGen/CXX/CXXCodeWriter.swift

```diff
@@ -0,0 +1,334 @@
+/// Class used to write the output CXX code from the given CXX AST.
```

Comment:
```suggestion
/// A type used to write the output CXX code from the given CXX AST.
```
Be careful, a class doesn't have the same semantics as a struct in Swift. Both terms are not interchangeable.